pycparser/_ast_gen.py

```diff
@@ -134,6 +134,22 @@ def _gen_children(self):
 
         return src
 
+    def _gen_eq(self):
```

Comment:
Is this related to the PR?

Reply:
Yes, required by tests.

Reply:
Could you please clarify which tests need this? pycparser has hundreds of AST tests which don't need this comparison, so I'm wondering what's new here. In any case I would prefer to decouple this large change from the `_Align*` PR.

Reply:
This one. I am fine to create a separate PR for that.

```
======================================================================
FAIL: test_alignment (tests.test_c_generator.TestCtoC)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/user/AppleDeveloper/pycparser/tests/test_c_generator.py", line 100, in test_alignment
    self._assert_ctoc_correct('_Alignas(32) int b;')
  File "/Users/user/AppleDeveloper/pycparser/tests/test_c_generator.py", line 79, in _assert_ctoc_correct
    "{!r} != {!r}".format(src, src2))
AssertionError: '_Alignas(32) int b;' != '_Alignas(32) int b;\n'
```

Reply:
Alright, let's move the equality stuff out to a separate PR that can be discussed after this one lands.